object_manager.go

```diff
@@ -316,3 +344,24 @@ func (objMgr *ObjectManager) CreateEADefinition(eadef EADefinition) (*EADefiniti
 
 	return newEadef, err
 }
+
+
+func (objMgr *ObjectManager) CreateMultiObjectRequest(body []*RequestBody) ([]map[string]interface{}, error){
```

Comment:
Okay I will change the name.

Actually CreateMultiObjectRequest does not implements IBObject interface because its does not have to implement SearchEA() method 

yeah makes sense to pass MultiRequest object. Thanks!